cmd/crank/xpkg/install.go

```diff
@@ -50,27 +50,44 @@ const (
 	errKubeClient    = "failed to create kube client"
 )
 
-// InstallCmd is exported so that it can be re-used by the beta xpkg subcommand.
+// installCmd installs a package.
+type installCmd struct {
+	// Arguments.
+	Kind    string `arg:"" help:"The kind of package to install. One of \"provider\", \"configuration\", or \"function\"." enum:"provider,configuration,function"`
+	Package string `arg:"" help:"The package to install."`
+	Name    string `arg:""  optional:"" help:"The name of the new package in the Crossplane API. Derived from the package repository and tag by default."`
+
+	// Flags. Keep sorted alphabetically.
+	RuntimeConfig        string        `placeholder:"NAME" help:"Install the package with a runtime configuration (for example a DeploymentRuntimeConfig)."`
+	ManualActivation     bool          `short:"m" help:"Require the new package's first revision to be manually activated."`
+	PackagePullSecrets   []string      `placeholder:"NAME" help:"A comma-separated list of secrets the package manager should use to pull the package from the registry."`
+	RevisionHistoryLimit int64         `short:"r" placeholder:"LIMIT" help:"How many package revisions may exist before the oldest revisions are deleted."`
+	Wait                 time.Duration `short:"w" default:"0s" help:"How long to wait for the package to install before returning. The command does not wait by default."`
+}
+
+func (c *installCmd) Help() string {
+	return `
+This command tells the Crossplane package manager to install a package. It uses
+~/.kube/config to connect to the package manager. You can override this using
+the KUBECONFIG environment variable.
```

Comment:
not really clear wording here, it doesn't connect to the package manager directly, atm it connects to kubernetes, so I would just be explicit for now and revisit if things changed down the line

Reply:
Right. I'm taking some inspiration from @plumbis here and saying something not strictly accurate. I don't think we really need to go into detail on how this works. What I'm trying to communicate is that unlike the other `xpkg` subcommands this one actually affects your control plane.

Reply:
Actually, this is probably better:

> This command installs a package into a Crossplane control plane. It uses ~/.kube/config to connect to the control plane. You can override this using the KUBECONFIG environment variable.

Updated.